internal/fs/inode/file.go

```diff
@@ -517,7 +522,10 @@ func (f *FileInode) SetMtime(
 
 	o, err := f.bucket.UpdateObject(ctx, req)
 	if err == nil {
-		f.src = storageutil.ConvertObjToMinObject(o)
+		minObjPtr := storageutil.ConvertObjToMinObject(o)
```

Comment:
you will need to use `var minObj gcs.MinObject` here similar to above, otherwise f.src may not be updated if minObjPtr is nil